src/parsers/power_system_table_data.jl

```diff
@@ -834,9 +876,11 @@ function _get_field_infos(data::PowerSystemTableData, category::InputCategory, d
     # The user's descriptors indicate that the raw data is already system-per-unit or not.
     per_unit = Dict{String, Bool}()
     unit = Dict{String,Union{String,Nothing}}()
+    descriptor_names = Vector{String}()
     for descriptor in data.descriptors[category]
         per_unit[descriptor["name"]] = get(descriptor, "system_per_unit", false)
         unit[descriptor["name"]] = get(descriptor, "unit", nothing)
+        push!(descriptor_names,descriptor["name"])
```

Comment:
Style guide says that there should be a space after comma.  Here and throughout the PR.